pkg/ingester/ingester.go

```diff
@@ -383,8 +424,13 @@ func (i *Ingester) Label(ctx context.Context, req *logproto.LabelRequest) (*logp
 	if err != nil {
 		return nil, err
 	}
+
+	maxLookBackPeriod := i.cfg.QueryStoreMaxLookBackPeriod
+	if i.store.ActivePeriodConfig().IndexType == shipper.BoltDBShipperType {
+		maxLookBackPeriod = time.Since(time.Now())
```

Comment:
i think this is probably a typo.  Not sure if the goal here is to disable maxLookBackPeriod, if so probably easier to just set it to zero :)

Reply:
sorry, my bad. It is a typo. It should be using the start time of the active schema.